.circleci/config.yml

```diff
@@ -167,6 +167,7 @@ jobs:
           name: Lint
           command: |
             $HOME/.local/bin/golangci-lint run -v \
+              --exclude cbor_gen.go \
```

Comment:
Please remove and rebase on master -- should resolve this

Reply:
oops, I meant to take that out anyway